src/Lit/Experimental.fs

```diff
@@ -0,0 +1,76 @@
+[<Experimental("This is an exploration space, anything can be added or removed in this module at any point")>]
+module Lit.Experimental
+
+
+open Fable.Core
+open Fable.Core.JsInterop
+open Lit
+
+[<AttachMembers>]
+type StateController<'T>(host, initial: 'T) =
+    inherit ReactiveController(host)
+    let mutable state = initial
+
+    member _.Value = state
+
+    member _.SetState(value: 'T) =
+        state <- value
+        host.requestUpdate ()
+
+[<AttachMembers>]
+type TwoArgsCtrl(host, initial: int, secondial: int) =
```

Comment:
don't mind the bad english names, I was just testing `System.ParamArray` and apply function arguments to the constructor